src/mk.rs

```diff
@@ -93,17 +93,29 @@ impl EncodingInfo for Multikey {
 
 impl From<Multikey> for Vec<u8> {
     fn from(val: Multikey) -> Self {
+impl Into<Vec<u8>> for Multikey {
+    fn into(self) -> Vec<u8> {
+impl From<Multikey> for Vec<u8> {
+    fn from(mk: Multikey) -> Vec<u8> {
         let mut v = Vec::default();
         // add in the sigil
         v.append(&mut SIGIL.into());
         // add in the key codec
         v.append(&mut val.codec.into());
+        v.append(&mut self.codec.clone().into());
+        v.append(&mut mk.codec.into());
         // add in the comment
         v.append(&mut Varbytes(val.comment.as_bytes().to_vec()).into());
+        v.append(&mut Varbytes(self.comment.as_bytes().to_vec()).into());
+        v.append(&mut Varbytes(mk.comment.as_bytes().to_vec()).into());
         // add in the number of codec-specific attributes
         v.append(&mut Varuint(val.attributes.len()).into());
+        v.append(&mut Varuint(self.attributes.len()).into());
+        v.append(&mut Varuint(mk.attributes.len()).into());
         // add in the codec-specific attributes
         val.attributes.iter().for_each(|(id, attr)| {
+        self.attributes.iter().for_each(|(id, attr)| {
+        mk.attributes.iter().for_each(|(id, attr)| {
```

Comment:
and this.

Reply:
I think you may be looking at an old version, Github has these marked as outdated and I don't see these issues on my branch.

I think I may have already resolved these, which is good because it means I'm not losing my mind

Reply:
Fixed as of https://github.com/cryptidtech/multikey/pull/2/commits/f479bd484b9c377a0a44684ec0b4160d04db1b71